nototools/fontdiff.py

```diff
@@ -63,43 +99,41 @@ def main():
                                      'pointed to by PATH_A and PATH_B.')
     parser.add_argument('path_a', metavar='PATH_A')
     parser.add_argument('path_b', metavar='PATH_B')
-    parser.add_argument('-t', '--diff_type', default='shape',
-                        help='type of comparison to run, "shape" or "gpos" '
-                        '(defaults to "shape"), if "gpos" is provided the '
-                        'input paths should point to ttxn output')
+    parser.add_argument('-t', '--diff-type', default='area',
```

Comment:
I think you can use 'choices' here to specify the available choices more directly.